setup.cfg

```diff
@@ -5,4 +5,4 @@ ignore = E203, W503
 exclude = **/snapshots/*.py, .venv
 
 [metadata]
-description-file = README.md
+description_file = README.md
```

Comment:
I don't understand this change :sweat: 

from [setup.cfg doc](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html) it seems the correct syntax is:
```
long_description = file: README.rst, CHANGELOG.rst, LICENSE.rst
```

Reply:
That was strange indeed. What was even stranger was that even modifying the file to something like `description_file = foo` or `long_description = bar` did not change anything: no build error and the generated package still contained the README.md. That's only when I removed the README.md file that I realized it was read from setup.py and set as `long_description` from there... :upside_down_face:

I just removed the "metadata" section from setup.cfg for now. Converting from setup.py to setup.cfg or pyproject.toml is out-of-scope for this PR.